src/main/java/net/greatstart/model/PasswordResetToken.java

```diff
@@ -0,0 +1,31 @@
+package net.greatstart.model;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.util.Date;
+
+@Data
+@Entity
+@Table(name = "password_reset_tokens")
+public class PasswordResetToken extends AbstractModel {
+
+    private static final int EXPIRATION = 60;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "EXPIRATION" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)